modules/user-access/pages/jwt-token.adoc

```diff
@@ -492,3 +492,63 @@ gadmin restart -y
 ====
 Existing environment variable may have some influence on the current use of TigerGraph.
 ====
+
+== Usage of GSQL JWT Token
+Starting from 4.1.0, the token generated by GSQL API is JWT Token and can be used on GSQL and RESTPP server for authentication.
+
+=== create JWT Token
+
+[console]
+----
+POST :14240/gsql/v1/tokens payload '{"secret":"<secret>", "graph":"<graph name>, lifetime: <number in ms>"}'
+----
+
+ * `secret`: the `secret` represents the user who owns so there is no need user login if secret set in the payload
```

Comment:
For more clarity - The secret represents the user. If a secret is provided in the payload, user login is not required. If no secret is specified in the payload, a secret will be automatically generated.